home/.config/iterm2/Iceberg.json

```diff
@@ -0,0 +1,405 @@
+{
```

Comment:
This is an exported JSON profile, how about to remove plist? It looks not updated... 🤔